### ISSUE TITLE
How can a CMS user remove the match field settings from a field?

### ISSUE THREAD

**Issue by joepagan** [verbb/formie#1288]
### Question

Hey, How can a CMS user remove the match field settings from a field?
It looks like there is no option to deselect if the field has been matched previously.

![image](https://user-images.githubusercontent.com/2337910/217822650-0be81efb-fbad-49b0-8eeb-9181c4f75387.png)

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.21",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.22](https://github.com/verbb/formie/releases/tag/2.0.22)
